Codemod to fix `math.isclose` comparison to `0` (#475)
pixee/codemodder-python: src/core_codemods/docs/pixee_python_fix-math-isclose.md

The default value for the `abs_tol` argument to a `math.isclose` call is `0`. Using this default when comparing a value against `0`, such as in `math.isclose(a, 0)` is equivalent to a strict equality check to `0`, which is not the intended use of the `math.isclose` function.

> This codemod adds `abs_tol=1e-09` to any call to `math.isclose` with one of of the first arguments evaluating to `0` if `abs_tol` is not already specified. `1e-09` is a starting point for you to consider depending on your calculation needs.

Reviewer: ```suggestion
This codemod adds `abs_tol=1e-09` to any call to `math.isclose` if one of of the first arguments evaluates to `0` and `abs_tol` is not already specified. `1e-09` is a suggestion which you may need to adjust depending on your calculation needs.
```
